base/logger/R/logger.R

```diff
@@ -20,6 +20,8 @@
 ##' Prints a debug message.
 ##' 
 ##' This function will print a debug message.
+##' This is a test change to check CI behavior.
```

Comment:
```suggestion
##' This is a test change to check CI behavior;
```
just to trigger a new workflow